src/legal-officer/transaction-protection/LocPublicDataForm.tsx

```diff
@@ -66,7 +66,7 @@ export default function LocPublicDataForm(props: Props) {
                                 message: 'The Data must contain at least 1 character'
                             },
                             maxLength: {
-                                value: 1000,
+                                value: 4096,
                                 message: 'The Data Name must contain at most 1000 characters'
```

Comment:
Message should should be changed accordingly. Also, from what I remember from the review meeting, it's not displayed, only the red ⚠️ shows - but I may be wrong.
```suggestion
                                message: 'The Data must contain at most 4096 characters'
```